### PR TITLE
Add shortcuts.md

### PR DESCRIPTION
**Pull request by vitvakatu**
### Pull Request Description

Brings back `shortcuts.md` that was removed when refactoring the repo. 

We plan to auto-generate this file from the list of our shortcuts, but for now, a manually edited and imperfect document is better than no document.

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [x] Unit tests have been written where possible.

### CODE REVIEW

**Review by hubertp — COMMENTED** [review]
Thank you!

File: app/gui2/shortcuts.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+## Graph Editor
+
+#### General Shortcuts
+
+| Shortcut                                                                          | Action                         |
+| --------------------------------------------------------------------------------- | ------------------------------ |
+| <kbd>Escape</kbd>                                                                 | Cancel current interaction     |
+| <kbd>Meta</kbd>+<kbd>`</kbd>                                                      | Show/hide Code Editor          |
+| <kbd>Meta</kbd>+<kbd>D</kbd>                                                      | Show/hide Documentation Editor |
```
Comment (AdRiley) [inline app/gui2/shortcuts.md:25]:
```suggestion
| <kbd>Meta</kbd>+<kbd>D</kbd>                                                      | Show/hide Documentation Editor |
| <kbd>Meta</kbd>+<kbd>,</kbd>                                                      | Show Settings                  |
| <kbd>Meta</kbd>+<kbd>/</kbd>                                                      | Show About Window              |
```

**Review by farmaazon — APPROVED** [review]
We may consider adding information about bindings.ts file, which should be always up-to-date.

**Comment by Frizi** [comment]
Looks really nice! We could later reuse those shortcut descriptions in the code, allowing us to present it to the users on an in-app help page (or keybindings editor, who knows).

**Review by Frizi — APPROVED** [review]

**Comment by jdunkerley** [comment]
This is great @vitvakatu. I'll copy it to the doc site too.

**Pull request merged**